common/core/keyboardprocessor/meson.build

```diff
@@ -6,7 +6,7 @@
 #
 
 project('keyboardprocessor', 'cpp', 'c',
-        version: '11.0.0',
+        version: '14.0.17',
```

Comment:
Too bad we can't get this version number from `VERSION.md`

Reply:
Ah this was inadvertently included. iirc, linux/scripts/[reconf.sh](https://github.com/keymanapp/keyman/blob/master/linux/scripts/reconf.sh#L59) parses version and replaces it in meson.build

Reply:
Ideally we have a meson.in -> meson.build so meson.build is not included in CI